### ISSUE TITLE
Error adding link with extended characters (UTF-8 URL encoding)

### ISSUE THREAD

**Issue by ethauvin** [SciDevs/delicious-api#24]
Seems like I'm always getting an error adding a link with a extended characters in the description or url parameters. For example:

curl 'https://user:passwd@api.del.icio.us/v1/posts/add?url=http%3A%2F%2Fen.wikipedia.org%2Fwiki%2FR%C3%A9sum%C3%A9&description=R%C3%A9sum%C3%A9'

Puzzling.

**Comment by tonioriol** [comment]
Same problem for me. Please help!

**Comment by ayavilevich** [comment]
me too

**Comment by peoplemerge** [comment]
Yes fixing this is not going to be too easy.  But it is on our roadmap this year.  Sorry for the trouble, please be patient, we'll keep you posted.

**Comment by lukasztaras** [comment]
Hi I have the same problem with cURL if i try to post below data i get error adding link while post appears on delicious.

https://api.del.icio.us/v1/posts/add?url=http%3A%2F%2F127.0.0.1%3A8080%2Fwordpress%2F2015%2F07%2F02%2Fsome+title%2F&description=some+description&tags=&extended=blahblahblah&shared=yes&replace=yes

**Comment by sumkincpp** [comment]
#43 seems to be relevant, issue still persists
